readme.md

```diff
@@ -3,4 +3,13 @@
     pip install -r requirements.txt
     python app.py
 
-App will be running on http://127.0.0.1:5000
\ No newline at end of file
+App will be running on http://127.0.0.1:5000
+
+This is a basic python app which requires Python 3.7 , SQLite3 and uses Flask as user 
```

Comment:
3.8 :wink:

Reply:
hahaha will change that thanks @NateBigStone

Reply:
👍 👍